### DISCUSSION TITLE
Package installment suggestion

### DISCUSSION THREAD

**Discussion by hongyingyue** [PaddlePaddle/PaddleSpatial#190] [BaiduKDDCup2022] [answered]
Dear host,

Could you please add below packages in tenforflow environment
- functools
- tensorflow-addons

Thanks

**Comment by ramber1836** [comment, marked as answer]
Working on it.

**Reply by ramber1836** [reply]
The package tensorflow-addons had been installed.
But, the package functools is incompatible with the current TensorFlow environment.
